data/hai/hai reveal 1 intro.txt

```diff
@@ -664,22 +685,19 @@ mission "Hai Leaks Response 2C"
 				has "met remington"
 			action
 				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`
-			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may have not heard of me before but I've sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis.`
+			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department."  He gestures to the woman behind him and says, "Let me introduce you to Terry."`
```

Comment:
I see where you want to go with removing his rambling, and it could be changed. But I wonder if we're losing some characterisation here. It's Remington's job to make things look good, and you can see he's starting to do his job. Then he decides to cut to the chase instead because he realises that this is a serious issue.

Reply:
Yes, this ramble probably needs to stay in. I'm not sure how to make it shorter without putting it back though.